docker/app/install-composer.sh

```diff
@@ -4,7 +4,7 @@
 # Origin: https://github.com/composer/composer/blob/master/doc/faqs/how-to-install-composer-programmatically.md
 # Licence: MIT
 
-EXPECTED_SIGNATURE=$(wget -q -O - https://composer.github.io/installer.sig)
```

Comment:
I must repeat this again. I don't see the point of us deviating from the official sample script here. The `.fetch-deps` do not end up in any of the image layers, so it has zero impact except for a very slightly longer build (does not make any practical difference).